.ddev/commands/host/matomo_init

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+## Description: Initialize Matomo basic environment
+## Usage: matomo:init
+## Example: ddev matomo:init
+
+#echo "Copy initial config file ..."
+#echo "@see https://developer.matomo.org/guides/tests-php#requirements"
+#ddev exec '\cp -r .ddev/initial-config/config.ini.php config/config.ini.php'
```

Comment:
I think it's ok to remove these, but we should call a bunch of `ddev matomo:console config:set` and set the config values from the initial ini config.

Reply:
As with several other changes this was not really intended, but a result of local modifications and a really bad `git add` :see_no_evil: 

```suggestion
echo "Copy initial config file ..."
echo "@see https://developer.matomo.org/guides/tests-php#requirements"
ddev exec '\cp -r .ddev/initial-config/config.ini.php config/config.ini.php'
```